src/tools/genpolicy/rules.rego

```diff
@@ -240,14 +261,9 @@ allow_by_anno(p_oci, i_oci, p_storages, i_storages) {
 allow_by_sandbox_name(p_oci, i_oci, p_storages, i_storages, s_name) {
     print("allow_by_sandbox_name: start")
 
-    s_namespace := "io.kubernetes.cri.sandbox-namespace"
-
-    p_namespace := p_oci.Annotations[s_namespace]
-    i_namespace := i_oci.Annotations[s_namespace]
-    print("allow_by_sandbox_name: p_namespace =", p_namespace, "i_namespace =", i_namespace)
-    p_namespace == i_namespace
+    i_namespace := i_oci.Annotations["io.kubernetes.cri.sandbox-namespace"]
 
-    allow_by_container_types(p_oci, i_oci, s_name, p_namespace)
+    allow_by_container_types(p_oci, i_oci, s_name, i_namespace)
```

Comment:
Now we need to get the namespace for the input for the case where we set the default namespace using `kubectl config set-context --current --namespace=kata-containers-k8s-tests` (this is what we do in the tests)